GTScript-Parallel-model.md

```diff
@@ -339,6 +339,7 @@ for k_ in range(k, K):
 ```
 
 The conclusion from this is that the user is not allowed to write to
-variables used with a horizontal offset either in the mask or elsewhere in
-the body of the while. The gtscript frontend implemented in gt4py contains checks
-to ensure that a user cannot write code incompatible with this restriction.
+fields in the body of the while loop that are used in the mask with
```

Comment:
```suggestion
fields in the body of the while loop that are used in the mask or elsewhere in the body with
```
I think this was actually correct and important, because as with the mask it would require a block-local halo as big as the extent*max number of iterations in the loop.